Extensions/TextInput/JsExtension.js

```diff
@@ -200,6 +209,30 @@ module.exports = {
         .setLabel(_('Width'))
         .setGroup(_('Border appearance'));
 
+      objectProperties
+        .getOrCreate('padding')
+        .setValue((objectContent.padding || 0).toString())
+        .setType('number')
+        .setLabel(_('Padding'))
+        .setGroup(_('Border appearance'));
+
+      objectProperties
+        .getOrCreate('maxLength')
+        .setValue(objectContent.maxLength || 20)
+        .setType('number')
+        .setLabel(_('Max length'))
```

Comment:
You can precise that's the Input value max length (and not the size of the input rectangle for instance)

Reply:
Should the default be 0 or -1? I probably don't want a max length by default on an input, that would be weird and unexpected, and not backward compatible?

Reply:
Oh yes that's true